Sources/MongoSwift/MongoClient.swift

```diff
@@ -344,18 +348,17 @@ public class MongoClient {
     }
 
     /**
-     * Run the `listDatabases` command.
+     * Retrieves a list of databases in this client's MongoDB deployment.
      *
      * - Parameters:
      *   - filter: Optional `Document` specifying a filter that the listed databases must pass. This filter can be based
      *     on the "name", "sizeOnDisk", "empty", or "shards" fields of the output.
      *   - session: Optional `ClientSession` to use when executing this command.
      *
-     * - Returns: An `EventLoopFuture<[DatabaseSpecification]>` containing the databases matching provided criteria.
-     *
-     * - Throws:
-     *   - `LogicError` if the provided session is inactive.
-     *   - `EncodingError` if an error is encountered while encoding the options to BSON.
+     * - Returns: An `EventLoopFuture<[DatabaseSpecification]>`. On success, the future contains an array of the
```

Comment:
According to the "Multiline Elements" section of Apple's [markup docs](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/MarkupSyntax.html), these blocks can begin on the next line down from their delimiters (i.e. `- Returns` in this case), and they also can be split into multiple paragraphs by inserting single empty lines between sections of text. I personally think its a little clearer if the return content starts on its own line below `- Returns`, and that the error information is separated into its own paragraph.

e.g.
https://github.com/mongodb/mongo-swift-driver/blob/763de6675ab45a7a0a5a7d3b39c99fcd0e12bb6c/Sources/MongoSwift/MongoCursor.swift#L241-L250

The way this is actually written likely could be improved, but I think the overall format is a bit clearer than a single paragraph. What do you think?

Reply:
sgtm, updated all of them